client/test/e2e/shared/components.js

```diff
@@ -32,3 +32,39 @@ exports.currencyInput = {
     return elm.getAttribute('ng-invalid');
   }
 };
+
+/**
+ * Object to help making end-to-end test,
+ * dateEditor.js.
+ * @public
+ */
+
+
+exports.dateEditor = {
```

Comment:
Could you update the patient registration e2e tests to use this wrapper?  That would provide a good usage example for others to follow.